### PR TITLE
Update installer

### PR DESCRIPTION
**Pull request by Err0r1**
Remove deprecated license
Update installer to accept only PHP 7.1
Rename FusionCMS to FusionGEN

### CODE REVIEW

File: install/system.php [hunk 1]
```diff
@@ -82,7 +82,14 @@ private function checkApacheModules()
     
     private function checkPhpVersion()
     {
-		die( version_compare(PHP_VERSION, '7.0', '>=') ? '1' : '0' );
+		$ver = phpversion();
+		if ($ver <= 7.0) {
```
Comment (mdjong1) [inline install/system.php:86]:
```suggestion
		if ($ver <= 7.0 || $ver >= 7.2) {
```

Comment (mdjong1) [reply install/system.php:86]: Merges the elseif into this

**Review by mdjong1 — CHANGES_REQUESTED** [review]

**Review by mdjong1 — APPROVED** [review]
Noice :octocat:

**Review by OMGhixD-OG — COMMENTED** [review]
Should ensure that less people attempt to create issue reports and or numerous amount of support related incidences on the discord server.

Good job!

**Review by OMGhixD-OG — APPROVED** [review]

**Pull request merged**